docs: Update node docs (#764)
grafana/phlare: docs/sources/configure-client/language-sdks/nodejs.md

          env: dev
```

> ## Sending data to Phlare with Pyroscope java integration

Reviewer: ```suggestion
## Sending data to Phlare with Pyroscope Node integration
```

To configure nodejs integration to send data to Phlare, replace the <URL> placeholder with the appropriate server URL. This could be the grafana.com Phlare URL or your own custom Phlare server URL.

> If you need to send data to grafana.com, you’ll have to configure HTTP Basic authentication. Replace <User> with your grafana.com stack user and <Password> with your grafana.com API key.

Reviewer: ```suggestion
If you need to send data to Grafana Cloud, you’ll have to configure HTTP Basic authentication. Replace `basicAuthUser` with your Grafana Cloud stack user ID and `basicAuthPassword` with your Grafana Cloud API key.
```

> If your Phlare server has multi-tenancy enabled, you’ll need to configure a tenant ID. Replace <TenantID> with your Phlare tenant ID.

Reviewer: ```suggestion
If your Pyroscope server has multi-tenancy enabled, you’ll need to configure a tenant ID. Replace `tenantID` with your Pyroscope tenant ID.
```
